openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Domains.spec.ts

```diff
@@ -1871,4 +1873,169 @@ test.describe('Domain Tree View Functionality', () => {
         .getByText(subDomain.data.displayName)
     ).toBeVisible();
   });
+
+  test('Verify Domain entity API calls do not include invalid domains field in glossary term assets', async ({
+    page,
+  }) => {
+    const { afterAction, apiContext } = await getApiContext(page);
+    const testClassification = new ClassificationClass({
+      provider: 'system',
+      mutuallyExclusive: false,
+    });
+    const testTag = new TagClass({
+      classification: testClassification.data.name,
+    });
```

Comment:
this test is only testing out on glossary, let's not have tags here, or else you can combine both the tests using sub step